### ISSUE TITLE
blockTime on Ganache UI mines 1tx per block

### ISSUE THREAD

**Issue by miguelhb** [trufflesuite/ganache#1359]
As i indicated in the tittle, i config Ganache UI to mine every 15 secs and make some tx. 

## Expected Behavior
I expected all transaction created in that time are mined in the same block.

## Current Behavior
Each block is mined each 15sec but only 1tx is included in one block. 
So, if i create 3 tx at the begguining, the first block includes the first tx, the second one the second tx and the 3rd one the last tx.

I use Ganache@2.0.4-beta on Linux.

**Comment by davidmurdoch** [comment]
Are your transactions very large (large contracts, maybe?)? If so, they just might not all fit in the block.

**Comment by miguelhb** [comment]
No, this is my smartcontract, it's to simple and conceptual. My transactions are calls to the methods setResult, getResult and the string of the parameter is short (one word)
```

pragma solidity ^0.5.0;

contract Gestion {
	string result = "sample"; // The result of the query is stored in this variable

  event customEvent (string mysqlcustom);

	function setResult(string memory input) public {
		result = input;
	}
	function getResult() public view returns(string memory)  {
		return result;
	}
	function CreateCustomEvent(string memory input) public {
		emit customEvent(input);
	}
}

```
I have seen this forum https://www.reddit.com/r/ethdev/comments/97br8o/ganache_blocktime/?sort=old where they haven't found the way to do it. Maybe it isnt possible?

**Comment by davidmurdoch** [comment]
Thanks for following up. Can you share the code that issues the transactions?

**Comment by miguelhb** [comment]
The transactions doesnt issue any code. But using Ganache, i have made some screenshots of two functions, setResult and CreateCustomEvent. As you can see Ganache include in different blocks, although it doesnt exceed the gas limit. I add the screenshots of blocks 6898 and 6899 and their transactions.
Block 6898
![Screenshot_20190718_110343](https://user-images.githubusercontent.com/36422996/61444884-91309680-a94c-11e9-8b5b-a575863322a5.png)
Transaction of block 6898
![Screenshot_20190718_110444](https://user-images.githubusercontent.com/36422996/61444932-a6a5c080-a94c-11e9-95eb-5bf63b546cba.png)
Block 6899 
![Screenshot_20190718_110326](https://user-images.githubusercontent.com/36422996/61444945-aefdfb80-a94c-11e9-81e9-52e871dbdc98.png)
Transaction of block 6899
![Screenshot_20190718_110503](https://user-images.githubusercontent.com/36422996/61444961-b58c7300-a94c-11e9-97f0-c8f9fc0286dd.png)
Besides, when i call the CreateCustomEvent function and the event is launched, the terminal prints the following JSON:
{ logIndex: 0,
  transactionIndex: 0,
  transactionHash: '0x9282b61744891020d4ebfc51de09cd52a6dda4ca6666c421efec578a0295b477',
  blockHash: '0x48b6bedc4f048c6a52c7f21f483d574a60e27d827b5d1fbc01638843daa27989',
  blockNumber: 6844,
  address: '0xEb6126a371FcD161481866Ed966FfacFdfC5f5CD',
  type: 'mined',
  id: 'log_0x8e7c2b8931acef29a8e136454b2a6d881fbfba6ea39eaa782dee705a0c797f1f',
  returnValues: { '0': 'a', mysqlcustom: 'a' },
  event: 'customEvent',
  signature: '0x818d2411fb91789e92f56372e7b3507b252d906389d61065b79c41fbb3df6169',
  raw: 
   { data: '0x000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000016100000000000000000000000000000000000000000000000000000000000000',
     topics: 
      [ '0x818d2411fb91789e92f56372e7b3507b252d906389d61065b79c41fbb3df6169' ] } }

**Comment by miguelhb** [comment]
I have done more tests. According to this link  [https://github.com/trufflesuite/ganache-cli/issues/108](url) , I have tried to stop de mining with miner_stop, tried to mine step by step my transactions with evm_mine and tried to mine  some transactions at the same time with miner_start. The problem persist, i'm not able to mine more than one transaction per block...
